javascript/exercises/cashRegister.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Javascript: The Cash Register</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <link href="../../common/css/bootstrap.css" rel="stylesheet">
+  </head>
+ <body>
+<div class="container">
+
+<h2 class="page-header">The Cash Register</h2>
+
+<p><em>  </em></p>
+<ul>
+ <li>Write a function called <code>cashRegister</code> that takes an object containing shopping cart values for various customers. The function should return a new object that contains the sum of each cart.
+
+ <li>Example input
+<pre>var prices = {
+  brenda: [{ 
+    "banana": {
+        "price": "$1.25"
+      }
+    },
+    { "handkerchief": {
+        "price": "$.99"
+      }
+    },
+    { "t-shirt": {
+        "price": "$25.01"
+      }
+    }],
+  claire: [
+    { "apple": {
+        "price": "$0.60"
+      }
+    },
+    { "nalgene": {
+        "price": "$10.34"
+      }
+    },
+    { "protein_shake": {
+        "price": "$22.36"
+      }
+    }]
+};
+
+// Solution
+
+// { "brenda": 33.3, "claire": 33.3 }
+</pre>
+
+<!-- <button class="btn" style="display:block;" onclick="showSolution(1)">See Solution</button>
+
+<br><br>
+<pre id="solution1" style="display:none;">
+function cashRegister(arr) {
+    var total = 0;   
+
+    for(var i = 0; i < arr.length; i++){
+      var price = arr[i].substr(1);
+      total += Number(price);
+    }
+    return total;
+}
+
+cashRegister(["$23.04", "$432.25", "$8.07"]);
```

Comment:
re-thinking this... can this be `cashRegister(prices)`? or maybe we can rename `prices` to `shoppingCarts` or something like that?